queries.sql

```diff
@@ -7,4 +7,14 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts > 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT MAX(escape_attempts) FROM animals;
+SELECT MAX(weight_kg) FROM animals WHERE species = 'digimon';
+SELECT MAX(weight_kg) FROM animals WHERE species = 'pokemon';
+SELECT MIN(weight_kg) FROM animals WHERE species = 'pokemon';
```

Comment:
* Kindly add queries that handle the transactions given in the [project requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=data.sql%20file.-,Inside%20a%20transaction%20update%20the%20animals%20table%20by%20setting%20the%20species,Take%20a%20screenshot%20of%20the%20results%20of%20your%20actions.,-Write%20queries%20to) as i can't see any queries for them